### PR TITLE
Add useful CIB tips for NodeUtilization

### PR DESCRIPTION
**Pull request by tahliar**
### PR creator: Description

Added some CIB details for NodeUtilization. TBH this whole section could use an overhaul, but I'll do that for SLE 16.

### PR creator: Are there any relevant issues/feature requests?

* bsc#1226835
* jsc#DOCTEAM-1475

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE-HA 15
  - [x] 15 next *(current `main`, no backport necessary)*
  - [x] 15 SP6
  - [x] 15 SP5
  - [x] 15 SP4
  - [ ] 15 SP3
  - [ ] 15 SP2
- SLE-HA 12
  - [ ] 12 SP5

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by tahliar** [comment]
image

File: xml/ha_resource_constraints.xml [hunk 1]
```diff
@@ -1241,6 +1241,16 @@
...
       op monitor timeout=30s interval=10s</command></screen>
     </example>
+    <para>
+     For either virtualization type, make sure to clone the <systemitem>NodeUtilization</systemitem>
+     resource so it can run on all nodes:
+    </para>
+<screen>&prompt.root;<command>crm configure cl_nu p_nu</command></screen>
+    <para>
+     We also recommend setting the <literal>placement-strategy</literal> property to
+     <literal>balanced</literal>:
```
Comment (zzhou1) [inline xml/ha_resource_constraints.xml:1251]: I'd like to rephrase as 

"""
The following command set the `placement-strategy` property to `balanced`.
"""

I personally have fewer recommendations for its value. The end user has their own free choice.

Comment (tahliar) [reply xml/ha_resource_constraints.xml:1251]: Thanks! That works for me :+1:

**Review by zzhou1 — APPROVED** [review]
THX!

**Review by dariavladykina — COMMENTED** [review]
LGTM!

**Pull request merged**